docs/wip/osm_configmap.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "OSM ConfigMap"
+description: "OSM ConfigMap"
+type: docs
+aliases: ["osm_configmap.md"]
+---
+
+Table of Contents
```

Comment:
Could the TOC be formatted similar to other docs for consistency?
eg: https://github.com/openservicemesh/osm/blob/main/tests/e2e/README.md
https://github.com/openservicemesh/osm/blob/main/README.md